examples/clip/clip/ckpt_transform.py

```diff
@@ -0,0 +1,24 @@
+import torch
+
+import mindspore as ms
+
+
+def pytorch_params(pth_file):
+    par_dict = torch.load(pth_file, map_location="cpu").state_dict()
+    pt_params = []
+    for name in par_dict:
+        parameter = par_dict[name]
+        if "ln_" in name:
+            name = name.replace(".weight", ".gamma").replace(".bias", ".beta")
+        elif name == "token_embedding.weight":
+            name = "token_embedding.embedding_table"
+        elif ".bn" in name or ".downsample.1." in name:
+            name = name.replace(".weight", ".gamma").replace(".bias", ".beta")
+            name = name.replace(".running_mean", ".moving_mean").replace(".running_var", ".moving_variance")
+        pt_params.append({"name": name, "data": ms.Tensor(parameter.numpy())})
+    return pt_params
+
+
+pth_path = "./ViT-B-32.pt"
+pt_param = pytorch_params(pth_path)
+ms.save_checkpoint(pt_param, "./ViT-B-32.ckpt")
```

Comment:
It's better to put them under `if __name__ == "__main__", and allow input args for different clip model version, e..g vit-h

Reply:
Great idea and have done now! ✅